scripts/deleteExpiredAccessTokens.js

```diff
@@ -0,0 +1,178 @@
+/*!
+Copyright 2019 OCAD University
+
+Licensed under the New BSD license. You may not use this file except in
+compliance with this License.
+
+You may obtain a copy of the License at
+https://github.com/GPII/universal/blob/master/LICENSE.txt
+*/
+
+// This script modifies the preferences database:
+// 1. Gather all the expired records of type "gpiiAppInstallationAuthorization" from the database,
+// 2. Delete them from the database,
+// A sample command that runs this script:
+// node deleteExpiredAccessTokens.js $COUCHDBURL [--deleteAll]
```

Comment:
Add comment to indicate COUCHDBURL is the url pointing to the GPII database. An example is `http://localhost:25984/gpii`.